src/services/StreamService.ts

```diff
@@ -28,7 +28,10 @@ class StreamService {
     // Create from a test result with multiple test types, multiple test result with one test type each
     const records: any[] = event.Records.filter((record: DynamoDBRecord) => {
       // Retrieve "INSERT" events
-      return record.eventName === "INSERT" || record.eventName === "MODIFY";
+      return (
+        record.eventName === "INSERT" ||
+        (record.eventName === "MODIFY" && process.env.PROCESS_MODIFY_EVENTS)
```

Comment:
Environment variables are explicitly stored as strings and so performing a truthy check on an environment variable is only checking whether the variable is defined. I would suggest creating a function like this and calling it.
```
function processModifyEvents(): boolean {
  if (process.env.PROCESS_MODIFY_EVENTS== 'false') return false;
  if (process.env.PROCESS_MODIFY_EVENTS== 'true') return true;

  throw Error('PROCESS_MODIFY_EVENTS environment variable must be true or false');
}
```

Reply:
Added